src/command_line_parser.cc

```diff
@@ -1443,6 +1461,43 @@ TritonParser::Usage()
   return ss.str();
 }
 
+std::tuple<std::string, std::string, std::string>
+TritonParser::ParseMetricsConfigOption(const std::string& arg)
+{
+  // Format is "<setting>=<value>" for generic configs/settings
+  int delim_name = arg.find(",");
+  int delim_setting = arg.find("=", delim_name + 1);
+
+  // No name-specific configs currently supported, though it may be in
```

Comment:
When would be name of the configuration group be useful?

Reply:
I don't have any specific ideas right now, but wanted to keep the format available in case. 

One general idea I can see coming in the near future is supporting OpenTelemetry vs Prometheus directly. In the event we keep old behavior and add OTLP on top of that, maybe you would configure each like:
```
--metrics-config otlp,key=value
--metrics-config prometheus,key=value
```

Or maybe groups of metrics could be configured differently:
```
--metrics-config compute,summaries=true
--metrics-config cache,histograms=true
```

Configuration groups aren't sorted out by any means, but I can see them having value as there have been a wide range of metrics related requests recently.

Reply:
@oandreeva-nv  for viz if openTelemetery Tracing settings
we must have cli options like:

```
--tracing-config <name>,<setting key>=<setting value>
```